policyengine_us/variables/gov/irs/income/taxable_income/deductions/qualified_business_income_deduction/qbid_amount.py

```diff
@@ -15,42 +15,81 @@ class qbid_amount(Variable):
     )
 
     def formula(person, period, parameters):
-        # computations follow logic in 2018 IRS Publication 535,
-        # Worksheet 12-A (and Schedule A for SSTB)
+
         p = parameters(period).gov.irs.deductions.qbi
-        # compute maximum QBID amount
+
+        # 1. Core inputs ---------------------------------------------------------
+        # IRS Form 8995, line 1 (qualified business income)
         qbi = person("qualified_business_income", period)
-        qbid_max = p.max.rate * qbi  # Worksheet 12-A, line 3
-        # compute caps
+        # W‑2 wages for the trade or business (Form 8995‑A, Part I)
         w2_wages = person("w2_wages_from_qualified_business", period)
-        b_property = person("unadjusted_basis_qualified_property", period)
-        wage_cap = w2_wages * p.max.w2_wages.rate  # Worksheet 12-A, line 5
-        alt_cap = (  # Worksheet 12-A, line 9
-            w2_wages * p.max.w2_wages.alt_rate
-            + b_property * p.max.business_property.rate
-        )
-        full_cap = max_(wage_cap, alt_cap)  # Worksheet 12-A, line 10
-        # compute phase-out ranges
-        taxinc_less_qbid = person.tax_unit("taxable_income_less_qbid", period)
+        # Unadjusted basis immediately after acquisition (UBIA)
+        # (Form 8995‑A, Part I)
+        ubia_property = person("unadjusted_basis_qualified_property", period)
+        # Specified service trade or business check box on Form 8995/8995‑A
+        is_sstb = person("business_is_sstb", period)
+        # Form 8995, line 6 (REIT/PTP income)
+        reit_ptp_income = person("qualified_reit_and_ptp_income", period)
+
+        # Form 1040, line 15 (taxable income) before QBID
+        taxable_income = person.tax_unit("taxable_income_less_qbid", period)
+        # Form 1040, filing status check box
         filing_status = person.tax_unit("filing_status", period)
-        po_start = p.phase_out.start[filing_status]
-        po_length = p.phase_out.length[filing_status]
-        # compute phase-out limited QBID amount
-        reduction_rate = min_(  # Worksheet 12-A, line 24; Schedule A, line 9
-            1, (max_(0, taxinc_less_qbid - po_start)) / po_length
+
+        threshold = p.phase_out.start[filing_status]
+        phase_in_range = p.phase_out.length[filing_status]
+
+        # 2. 20 % of QBI ---------------------------------------------------------
+        qbid_max = p.max.rate * qbi
+        # 20% of qualified business income (Worksheet 12‑A)
```

Comment:
Can we remove specific parameter values from the comments? If e.g. the 20% rate changes these would be outdated and confusing if reviewing the logic in a couple of weeks / months / years